engine/src/main/java/org/hibernate/validator/internal/engine/ValidatorFactoryConfigurationHelper.java

```diff
@@ -138,39 +138,39 @@ static boolean checkPropertiesForBoolean(Map<String, String> properties, String
 		return contributors;
 	}
 
-	static boolean getAllowParallelMethodsDefineParameterConstraints(ConfigurationImpl hibernateSpecificConfig, Map<String, String> properties) {
+	static boolean getAllowParallelMethodsDefineParameterConstraints(AbstractConfigurationImpl<?> hibernateSpecificConfig, Map<String, String> properties) {
```

Comment:
I didn't think about it as I wanted a 1-1 change.

We might do that in the future if we think it makes sense.

(Frankly, I'm more to avoid adding new methods if they are not strictly necessary)

Reply:
If the methods are not exposed in the interfaces, only in the impl, and not exposed anywhere, we can probably remove them.